keel-retrofit/keel-retrofit.gradle

```diff
@@ -17,11 +17,11 @@ apply plugin: "java-library"
 apply plugin: "kotlin-spring"
 
 dependencies {
-  api "com.squareup.retrofit2:retrofit:${spinnaker.version('retrofit2')}"
-  api "com.squareup.retrofit2:converter-jackson:${spinnaker.version('retrofit2')}"
+  api "com.squareup.retrofit2:retrofit"
+  api "com.squareup.retrofit2:converter-jackson"
   api "com.jakewharton.retrofit:retrofit2-kotlin-coroutines-adapter:0.9.2"
```

Comment:
Left this version out of the bom as we shouldn't need it for long (once the next release of Retrofit2 happens we can drop it).